### PR TITLE
use qunit css instead of inline styles

### PR DESCRIPTION
**Pull request by asapach**
As discussed in #109 this is what it would look like if we got rid of inline styles for `qunit-fixture` and used QUnit CSS instead.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "lint": "grunt eslint",
-    "test": "grunt",
+    "test": "grunt && mocha test/lib",
```
Comment (Johann-S) [inline package.json]: `mocha test/lib` is already launched by travis, by doing that you launched it twice

**Comment by asapach** [comment]
@Johann-S, but it doesn't run locally unless I run it explicitly - is that expected? Maybe just use `npm/yarn test`?

**Comment by Johann-S** [comment]
yep that's not convenient we should do that in a npm script to enable folks to use it 🤔

**Comment by Johann-S** [comment]
Good job @asapach 👍

**Pull request merged**